From abb17038b2819f10c2baedc47467f1306356d806 Mon Sep 17 00:00:00 2001
From: Jin Wei <wei.a.jin@intel.com>
Date: Tue, 14 May 2013 13:27:46 +0800
Subject: [PDK MR2 Temp] fix pdk build error

BZ: 117199

Revert "libcamera2: use -Werror for R&D builds"
This reverts commit c4acb5daacf8eca7bc7f80ca5113ca4d5f94f539.

Change-Id: I64eab706aee3acd6cbd9a905e543ad215f7fcc43
---
 Android.mk |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Android.mk b/Android.mk
index a506b02..f4b9f78 100644
--- a/Android.mk
+++ b/Android.mk
@@ -148,7 +148,7 @@ endif
 
 # enable R&D features only in R&D builds
 ifneq ($(filter userdebug eng tests, $(TARGET_BUILD_VARIANT)),)
-LOCAL_CFLAGS += -DLIBCAMERA_RD_FEATURES -Wunused-variable -Werror
+LOCAL_CFLAGS += -DLIBCAMERA_RD_FEATURES -Wunused-variable
 endif
 
 # The camera.<TARGET_DEVICE>.so will be built for each platform
-- 
1.7.4.1

